### PR TITLE
Update TLA+ docs page

### PR DESCRIPTION
**Pull request by heidihoward**
Just a quick tidy up of the TLA+ docs page

### CODE REVIEW

File: doc/architecture/raft_tla.rst [hunk 1]
```diff
@@ -1,44 +1,37 @@
...
 
-.. note::  During development, it helps to use simulation mode which performs a depth-first search of the search tree (instead of the default breadth first that is very slow). Turn on the simulation mode with ``-simulate -depth 100000`` (using a very large number as a maximum depth). Note that this mode never completes (but will find errors in minutes instead of hours).
+.. tip::  During development, it helps to use simulation mode which performs a depth-first search of the search tree (instead of the default breadth first that is very slow). Turn on the simulation mode with ``-simulate -depth 100000`` (using a very large number as a maximum depth). Note that this mode never completes (but will find errors in minutes instead of hours).
```
Comment (lemmy) [inline doc/architecture/raft_tla.rst:37]: `100.000` is rather excessive.  I've been using values in the range 100..500.

**Review by lemmy — APPROVED** [review]

File: doc/architecture/raft_tla.rst [hunk 2]
```diff
@@ -1,44 +1,37 @@
...
 
-The first purpose can be achieved by running the specification with a model checker. While we used `TLC <http://lamport.azurewebsites.net/tla/tools.html>`_ which comes built-in with the TLA+ tools, any model checker that works with TLA+ should be possible to make run.
-
-The second purpose is achieved simply with the TLA+ code. However, it is important to understand that the **TLA+ specification has no binding to the actual implementation**. This means that any change in the Raft implementation after the last modification of this specification may not be reflected and can still contain unexpected edge cases. However, comments in the TLA+ code are meant to help make this transition easier. Overall, we expect the CI tests in addition to this model checking to give us a good coverage of the behaviors to expect.
```
Comment (lemmy) [inline doc/architecture/raft_tla.rst:13]: Feels good to remove "**TLA+ specification has no binding to the actual implementation**". :-)

File: doc/architecture/raft_tla.rst [hunk 3]
```diff
@@ -1,44 +1,37 @@
-TLA+ Specification
-==================
+TLA+ Specifications
+===================
 
-The CCF repository includes a formal specification in TLA+ of CCF's consensus algorithm, which is a variant of Raft.
+The CCF repository includes two formal specification in TLA+ of CCF:
 
-This specification serves two purposes:
+* :ccf_repo:`tla/consistency` which models CCF at a high level of abstract to test the consistency model exposed to the clients, and
```
Comment (achamayou) [inline doc/architecture/raft_tla.rst:6]:
```suggestion
* :ccf_repo:`tla/consistency` which models CCF at a high level of abstraction to test the consistency model exposed to the clients, and 
```

File: doc/architecture/raft_tla.rst [hunk 4]
```diff
@@ -1,44 +1,37 @@
...
 
-This specification serves two purposes:
+* :ccf_repo:`tla/consistency` which models CCF at a high level of abstract to test the consistency model exposed to the clients, and 
+* :ccf_repo:`tla/consensus` which models in detail the custom distributed consensus protocol implemented in CCF.
 
-1. To catch problematic edge cases in our version of the Raft protocol that may not be trivially found or that may not be covered by tests.
-2. To serve as a formal future-proof specification of the protocol as it is intended to function.
+CCF implements various modifications to Raft as it was originally proposed by Ongaro and Ousterhout. Specifically, CCF constrains that only appended entries that were *signed* by the primary can be committed. Any other entry that has *not* been signed is rolled back. Additionally, the CCF implementation introduced a variant of the reconfiguration that is different from the one proposed by the original Raft paper. By default, reconfigurations are done via one transaction (as described :doc:`here </architecture/consensus/1tx-reconfig>`). The TLA+ consensus specification models the intended behavior of Raft as it is modified for CCF.
```
Comment (achamayou) [inline doc/architecture/raft_tla.rst:9]: Can we remove "By default"? 2tx has been completely removed, and there is not intention to bring it back.

**Review by achamayou — APPROVED** [review]

File: doc/architecture/raft_tla.rst (hunk 4 above)
Comment (heidihoward) [reply doc/architecture/raft_tla.rst]: done in https://github.com/microsoft/CCF/pull/5753/commits/de75b5408d43bfe56690244fe12a0333ada1d082
